### PR TITLE
GPU-enabled Point Cloud Densification (OpenMVS)

### PR DESCRIPTION
**Pull request by pierotofy**
Brings the latest changes from OpenMVS that enable GPU-accelerated point cloud densification.

Still needs testing and some issues to be fixed.

### CODE REVIEW

File: opendm/osfm.py [hunk 1]
```diff
@@ -214,8 +214,8 @@ def setup(self, args, images_path, reconstruction, append_config = [], rerun=Fal
                 "bruteforce": "BRUTEFORCE"
             }
 
-            if not has_gps:
-                log.ODM_INFO("No GPS information, using BOW matching")
+            if not has_gps and not 'matcher_type_is_set' in args:
+                log.ODM_INFO("No GPS information, using BOW matching by default (you can override this by setting --matcher-type explicitely)")
```
Comment (Saijin-Naib) [inline opendm/osfm.py]: Minor typo.

explicitely should read explicitly.

Comment (pierotofy) [reply opendm/osfm.py]: Uups! Thanks!

**Comment by pierotofy** [comment]
Brighton beach speed up on densification:

image

**Comment by pierotofy** [comment]
Sheffield cross:

image

**Comment by Saijin-Naib** [comment]
To say that this is impressive would be a massive understatement and selling this short.

Wow. Just awesome stuff!

**Pull request merged**